### ISSUE TITLE
Aggregate multiple values from a same header name

### ISSUE THREAD

**Issue by Chavjoh** [uniVocity/univocity-parsers#188]
Hello,

Here's a useful enhancement: 
Be able to load all the values from the same column name into an array in a bean.

For example, we have this CSV file:
```
id,field,field
1,value1,value2
```
And we would like to have ```value1``` and ```value2``` in ```String[] values```:
```java
@Parsed(field = "field")
private String[] values;
```

Thank you in advance !
Best Regards.

**Comment by jbax** [comment]
Thanks for the suggestion. I'll try to make this work when the time allows.

**Comment by jbax** [comment]
Done, the latest released 2.6.0-SNAPSHOT version already includes this enhancement.

You must use this on an annotated method to accumulate the values like you did. If a simple attribute is mapped to header that appears multiple times, it will take the value under the last column (all values of the repeated columns will be set, sequentially overriding the previous value).

Arrays are not supported.

Thank you for using our parsers!
